MAUI/TreeMap/migration.md

```diff
@@ -0,0 +1,1450 @@
+        </treemap:SfTreeMap.BindingContext>
+            <treemap:SfTreeMap.LeafItemSettings>
+                <treemap:TreeMapLeafItemSettings LabelPath="Country">
+                    <treemap:SfTreeMap.LeafItemBrushSettings>
+                    <treemap:TreeMapUniformBrushSettings Brush="#F8D7B9"/>
+                    </treemap:SfTreeMap.LeafItemBrushSettings>
+            </treemap:TreeMapLeafItemSettings>
+        </treemap:SfTreeMap.LeafItemSettings>
+        <treemap:SfTreeMap.Levels>
+            <treemap:TreeMapLevel GroupPath="Continent"/>
+        </treemap:SfTreeMap.Levels>
+    </treemap:SfTreeMap>
+</ContentPage>
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.TreeMap;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfTreeMap treeMap = new SfTreeMap();
+        PopulationViewModel viewModel = new PopulationViewModel();
+        treeMap.DataSource = viewModel.PopulationDetails;
+        this.treeMap.Levels.Add(new TreeMapLevel() { GroupPath = "Continent"});
+        this.Content = treeMap;
+    }
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="ViewModel.cs" %}
+
+public class PopulationViewModel
+{
+    public PopulationViewModel()
+    {
+        this.PopulationDetails = new ObservableCollection<PopulationDetails>()
+            {
+                new PopulationDetails() { Continent ="North America", Country = "United States of America", Population = 339996564 },
+                new PopulationDetails() { Continent ="North America", Country = "Mexico", Population = 128455567 },
+                new PopulationDetails() { Continent ="North America", Country = "Canada", Population = 38781292 },
+                new PopulationDetails() { Continent ="South America", Country = "Bolivia", Population = 12388571 },
+                new PopulationDetails() { Continent ="North America", Country = "Costa Rica", Population = 5212173 },
+                new PopulationDetails() { Continent ="South America", Country = "Uruguay", Population = 3423109 },
+            };
+    }
+    public ObservableCollection<PopulationDetails> PopulationDetails
+    {
+        get;
+        set;
+    }
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="Model.cs" %}
+
+public class PopulationDetails
+{
+    public string Country { get; set; }
+    public string Continent { get; set; }
+    public int Population { get; set; }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<table>
+<tr>
+<th>Xamarin SfTreeMap control</th>
+<th>.NET MAUI SfTreeMap control</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[LabelPath](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.TreeMapLevel.html#Syncfusion_SfTreeMap_XForms_TreeMapLevel_LabelPath)'| markdownify }}</td>
+<td>GroupPath</td>
+<td>Gets or sets the Header path property. This property is used to set the path for header of the tree map.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
```

Comment:
Remove null. Property is avail in XForms also.

Refer link - https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.TreeMapFlatLevel.html#Syncfusion_SfTreeMap_XForms_TreeMapFlatLevel_GroupBackground

https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.TreeMapHierarchicalLevel.html#Syncfusion_SfTreeMap_XForms_TreeMapHierarchicalLevel_ChildBackground

For better clarification - you can refer our API document link - https://syncfusion.sharepoint.com/sites/MAUI/_layouts/15/Doc.aspx?sourcedoc=%7B8C26BFDD-91D7-44F5-92D5-EA61FE8FF175%7D&file=849816%20TreeMap%20control%20API%20.docx&action=default&mobileredirect=true&CT=1710044739678&OR=ItemsView

Reply:
Added related API